sql/updates/2.3.0--2.3.1.sql

```diff
@@ -0,0 +1,5 @@
+CREATE SCHEMA IF NOT EXISTS timescaledb_experimental;
```

Comment:
I thought timescaledb_experimental is not part of 2.3.1

Reply:
Great catch @gayyappan! Thank you. :)